deeppavlov/models/go_bot/launch_train_bot.py

```diff
@@ -0,0 +1,163 @@
+
+    gobot_config = read_json(configs.go_bot.gobot_simple_dstc2)
+    gobot_config['chainer']['pipe'][-1]['embedder'] = None
+
+    db_path = 'db.sqlite'
+    gobot_config['chainer']['pipe'][-1]['database'] = {
+        'class_name': 'sqlite_database',
+        'primary_keys': ['name'],
+        'save_path': os.path.join(bot_data_dir, db_path)
+    }
+
+    gobot_config['chainer']['pipe'][-1]['slot_filler']['config_path'] = path_to_slotfill_config
+
+    slot_names_dstc2 = ['pricerange', 'this', 'area', 'food']
+
+    gobot_config['chainer']['pipe'][-1]['tracker']['slot_names'] = slot_names_dstc2
+
+    gobot_config['chainer']['pipe'][-1]['template_type'] = 'DefaultTemplate'
+    gobot_config['chainer']['pipe'][-1]['template_path'] = os.path.join(bot_data_dir, 'simple-dstc2-templates.txt')
+
+    gobot_config['metadata']['variables']['DATA_PATH'] = bot_data_dir
+    gobot_config['metadata']['variables']['MODEL_PATH'] = bot_model_dir
+
+    gobot_config['train']['epochs'] = 200
+    gobot_config['train']['batch_size'] = 8
+    gobot_config['train']['max_batches'] = 250
+    gobot_config['train']['log_on_k_batches'] = 20
+    gobot_config['train']['val_every_n_batches'] = 40
+    gobot_config['train']['log_every_n_batches'] = 40
+
+    path_to_gobot_config = os.path.join(bot_model_dir, 'gobot_config.json')
+    with open(path_to_gobot_config, 'w') as f:
+        json.dump(gobot_config, f, indent=2)
+        log.info(f'gobot config is saved to {path_to_gobot_config}')
+
+    return gobot_config
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    # Required parameters
+    parser.add_argument("--bot_data_dir", default=None, type=str, required=True,
+                        help="Relative path from working directory to a directory where go-bot data will be saved")
+    parser.add_argument("--bot_model_dir", default=None, type=str, required=True,
+                        help="Relative path from working directory to a directory where go-bot model will be saved")
+
+    # Other parameters
+    parser.add_argument("--working_directory", default="", type=str, required=False,
+                        help="Directory from which relative paths are built")
+    parser.add_argument("--load_data", action='store_true',
+                        help="Whether to load data")
+    parser.add_argument("--do_train", action='store_true',
+                        help="Whether to train the bot")
+    parser.add_argument("--do_eval", action='store_true',
+                        help="Whether to evaluate the bot performance.")
+    parser.add_argument("--max_train_data", default=None, type=int, required=False,
+                        help="Maximum number of dialogues to take from training data")
+
+    args = parser.parse_args()
+
+    bot_data_dir = os.path.join(args.working_directory, args.bot_data_dir)
+    bot_model_dir = os.path.join(args.working_directory, args.bot_model_dir)
```

Comment:
I highly recommend you to look at the [pathlib](https://docs.python.org/3.7/library/pathlib.html) module. It's what we choose to use instead of `os.path` in the library code.